apps/omg_api/lib/application.ex

```diff
@@ -51,26 +51,23 @@ defmodule OMG.API.Application do
         service_name: :in_flight_exit,
         synced_height_update_key: :last_in_flight_exit_eth_height,
         get_events_callback: &OMG.Eth.RootChain.get_in_flight_exit_starts/2,
-        process_events_callback: &ignore_validities/1
+        process_events_callback: &(&1 |> OMG.API.State.exit_utxos() |> ignore_validities())
       ),
       OMG.API.EthereumEventListener.prepare_child(
         service_name: :piggyback,
         synced_height_update_key: :last_piggyback_exit_eth_height,
         get_events_callback: &OMG.Eth.RootChain.get_piggybacks/2,
-        process_events_callback: &ignore_validities/1
+        process_events_callback: &(&1 |> OMG.API.State.exit_utxos() |> ignore_validities())
```

Comment:
I think we can leave  State.exit_utxos call inside ignore_validities. They both take one argument. If we want to make exit_utxos call more explicit we can rename ignore_validities function instead.

Reply:
yes, I did it this way because the function clearly was misnamed. However, I think I'm slightly inclined towards the non-DRY but explicit formulation, hmmm... but ok, let's do what you suggest